test/spec-tests/cql/CqlIntervalOperatorsTest.cql

```diff
@@ -515,11 +513,9 @@ define "Except": Tuple{
 
 define "In": Tuple{
   "TestInNull": Tuple{
-    skipped: 'Wrong answer (true vs false)'
```

Comment:
This is the test I think we need to continue ignoring but check in w/ Bryn about because I think the answer _should_ be true.

Reply:
As I noted above, Bryn confirmed that the test is wrong.  Can you please fix it according to the details in my other comment?

Reply:
I am working on making this change; does anything need to be changed in the `CQLIntervalOperatorsTest.xml` file, or should this stay true to the contents of the spec?

Reply:
Upon further testing, I've come to learn that the XML file is the one that requires the change in order for the build of CQL and ELM to work! Updating that accordingly, though I don't know if we need to be concerned about this file diverging from the spec.

Reply:
Right.  While we want to try to stay as close to the spec test files as possible, we've already made modifications where tests are blatantly wrong.  In those cases, I've also submitted a pull request to the CQL spec repo with the same changes (so if/when that PR is accepted, we'll be in alignment again).

For this specific change, I did not submit a PR to the CQL spec repo, but Bryn said he would fix it himself.